uniswap/model/parts/uniswap_model.py

```diff
@@ -147,4 +172,10 @@ def getInputPrice(input_amount, input_reserve, output_reserve, _params):
     input_amount_with_fee = input_amount * fee_numerator
     numerator = input_amount_with_fee * output_reserve
     denominator = (input_reserve * fee_denominator) + input_amount_with_fee
-    return int(numerator // denominator)
\ No newline at end of file
+    return int(numerator // denominator)
```

Comment:
Does floor division operator already return an integer, doesn't?

Reply:
Apparently it does, but Markus said it's a convenience to be "redundant" and use both int function and floor divison operator so that it works appropriately (don't really know why lol)